cgi-bin/npg_qc

```diff
@@ -1,18 +1,20 @@
 #!/usr/local/bin/perl -T
+use strict;
+use warnings;
 
 my @npg_libs;
 BEGIN {
   my $libs_string = $ENV{'NPG_PERL5LIB'};
   if ($libs_string) {
     my @libs = split /:/smx, $libs_string;
     foreach my $l (@libs) {
-      my ($dtl) = $l =~ /([a-z0-9\/\._\-]+)/i;
+      my ($dtl) = $l =~ /([\w\/\.\-]+)/smxia;
+      #'a' modifiler for ascii only interpretation of \w
       if ($dtl) {push @npg_libs, $dtl;}
     }
   }
 }
 
-use strict;
 use lib @npg_libs;
 use warnings;
```

Comment:
If we end up having import of strict and warnings modules at the top of the page, this line shoudl be deleted.